packages/Kokkos/src/DTK_PointCloud_def.hpp

```diff
@@ -228,7 +227,7 @@ Kokkos::Array<SC, 6> PointCloud<SC, LO, GO, NO>::globalBoundingBox() const
  * \return The point global ids. Dimensions: (Point)
  */
 template <class SC, class LO, class GO, class NO>
-const auto PointCloud<SC, LO, GO, NO>::globalIds() const -> const global_id_view
+auto PointCloud<SC, LO, GO, NO>::globalIds() const -> const global_id_view
```

Comment:
That's funny that this passed with gcc. clang throws an error: 

error: function with trailing return type must specify return type 'auto', not 'const auto'
const auto PointCloud<SC, LO, GO, NO>::coordinates() const